app/views/results/marker/_keybinding.js.erb

```diff
@@ -69,10 +69,25 @@ Mousetrap.bind('down', function(e) {
   }
 });
 
-// when enter is pressed, and there is an active rubric selected, trigger select and collapse
+// When enter is pressed, and there is an active rubric selected, trigger select and collapse
 Mousetrap.bind('enter', function(e) {
   if ($('.active-rubric').length) {
     e.preventDefault();
     $('.active-rubric').trigger('onclick');
   }
 });
+
+// Press shift+n for new annotation modal to appear
+Mousetrap.bind('shift+n', function(e) {
+  if ($('#annotation_dialog:visible').length == 0) {
+    make_new_annotation();
+  }
+});
+
+// When shift+enter is pressed and the annotation modal is open, submit it
+Mousetrap.bind('shift+enter', function(e) {
```

Comment:
shift + enter is sometimes used in text editors to insert a soft line break, so I don't think it's a good choice here. If you use the `mod + enter` I believe Mousetrap will turn that into ctrl/command + enter, which is preferable.